File: Codeforces Checking 1791A.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin>>t;
    while(t--) {
        char c;
        cin>>c;
        bool yes=false;
        string s="codefrs";
        for(auto x:s) {
           if(x==c) {
              yes=true;
              break;
           }
        }
        yes?cout<<"YES\n":cout<<"NO\n";
    }
}

File: Codeforces Checking 1791A.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		for(int t=sc.nextInt();t>0;t--) {
			String c=sc.next();
			if("codefrs".contains(c)) {
				System.out.println("YES\n");
			}
			else System.out.println("NO\n");
		}
	}
}